### DISCUSSION TITLE
[Question][Hw0_2] Problem 4

### DISCUSSION THREAD

**Discussion by woojinnn** [prosyslab-classroom/cs424-program-reasoning#379] [Q&A]
In `satTest.ml` file, sample answer for `solve (And (Or (Z, Not Y), Or (Not X, Not Z)))` is `SAT (false, false, false)`.  
However, `SAT (true, false, false)` also makes sense.  
Is there specific reason that the answer should be `(false, false, false)`?

**Comment by sms0509** [comment]
I also have the same problem. I got 'All checks have failed' message from github when I pushed my modified code to the private repository.

**Comment by KihongHeo** [comment]
Hi. You folks are right. There are several possible solutions. The "expected one" is just one option.
